modules/platform/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/TestTreeCommentNotification.java

```diff
@@ -94,4 +98,16 @@ public void testAutoSubscribingOnlyOnceToNewComments() {
         assertTrue(subscriptions.isEmpty());
     }
 
+    @Test
+    public void testCommentsStructureOnDocumentWithoutDublincore() {
+        DocumentModel dublincoreLess = session.createDocumentModel("File");
+        dublincoreLess.putContextData("disableDublinCoreListener", Boolean.TRUE);
+        dublincoreLess = session.createDocument(dublincoreLess);
+        assertNull(dublincoreLess.getPropertyValue("dc:contributors"));
+        transactionalFeature.nextTransaction();
+        Comment comment = commentManager.createComment(session,
+                newComment(dublincoreLess.getId(), "Test message again"));
+        assertNotNull(comment);
```

Comment:
Worth checking that the author is `null`, right? As it comes from `dc:contributors`.

Reply:
The author of the comment should be the session principal. It's not really the purpose of this test so I don't think we should check that here.